vtex/omnishipping
# CHK-129: Fix address form fields visible even for an invalid postal code

#### What is the purpose of this pull request?

This PR fixes the scenario where the user would see the address form fields even if it was an invalid postal code.

#### How should this be manually tested?

- Get to this [workspace](https://mandatoryaddress--vtexgame1.myvtex.com/checkout/cart/add/?sku=282&qty=1&seller=1&sc=1)
- Proceed to checkout
- `test@mailinator.com`
- Click on "Entregar em outro endereço"
- Type any digit on the postal code input
- You should not see the address form fields right away
- Type `22230-001` as postal code
- Now you should see the address form fields (autocompleted or not)
 
#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/12574426/86900148-5d142f80-c0e1-11ea-83a1-c4b9373a542b.png)

#### Types of changes
